app/models/environment_variable.rb

```diff
@@ -5,4 +5,31 @@ class EnvironmentVariable < ApplicationRecord
   validates :value, presence: true
 
   belongs_to :bridge
+
+  before_save :encrypt, if: proc { |c| c.value_changed? }
+
+  delegate :encrypt_and_sign, :decrypt_and_verify, to: :encryptor
+
+  KEY = ActiveSupport::KeyGenerator.new(
```

Comment:
Does this generate a new key each time we boot up or would it generate the same key? I am worried if we turn on the server, make a env var, turn off, turn back on, that new KEY can still decrypt the env var we made on the first boot up.